src/pages/home/home.module.css

```diff
@@ -0,0 +1,82 @@
+
+main{
+    padding: 0;
+}
+
+section{
+    padding: var(--padding-fluid-lg);
+}
+
+.hero{
+    position: relative;
+    height: 80vh;
+    background-image: var(--bg-gradient-overlay);
+    background-size: cover;
+    background-attachment: fixed;
+}
+/* 
+.hero::before{
+    content: '';
+    position: absolute;
+    inset: 0;
+    background-color: var(--tr-gray-overlay);
+    background-image: var(--bg-gradient-overlay);
+}
+
+.hero >*{
+    position: relative;
+    z-index: 1;
+} */
+
+.heroHeading{
+    font-size: clamp(2rem, 10vw - 1rem, 3rem);
+    color: var(--tr-background);
+    max-width: 30ch;
+}
+
+.cta{
+    font-size: 1.2rem;
+
```

Comment:
try to remove space

Reply:
Thanks for pointing it out, I have fixed it in the latest commit